client/posts/posting/view.ts

```diff
@@ -215,6 +215,22 @@ export default class FormView extends PostView {
         this.input.value = this.input.value.slice(0, -length)
     }
 
+    // Color input
+    public colorInput() {
+        this.el.
+            querySelector(".post-container").
+            querySelector("blockquote").
+            classList.add("red")
```

Comment:
instead of add you can use [toggle](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Methods) and pass a single boolean, that way you don't need two methods.